### DISCUSSION TITLE
Gherkin world

### DISCUSSION THREAD

**Discussion by jonsamwell** [codeceptjs/CodeceptJS#2979] [Q&A]
Hi all is there any way to pass a Gherkin world object when running with cucumber tests?

**Comment by cbruyndoncx** [comment]
With codeceptjs 3.2.2 the @ is illegal character and errors out.

I have solved it using a global variables outside the GWT steps.
Your world object gives me the interesting idea to have a global world object variable and just tag the detailed variable eg world.email, world.user etc
